### ISSUE TITLE
Info pro cizince – jak se dostat na místo

### ISSUE THREAD

**Issue by honzajavorek** [pyvec/pyvo.cz#17]

**Comment by kvbik** [comment]
good idea. prazsky meetup ma sraz na schodech venku u vylezu z metra B Andel. bohuzel nam zrusili ten apple obchod, takze se to misto uz spatne popisuje.

**Comment by torsava** [comment]
Venues už teď mají mimo pole `address` také `notes`, kde se tato informace dá napsat. Ale pole `notes` není v současné chvíli možné poskytnout v bilingvní formě — toto by stálo za to doimplementovat.
